File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/codex-sl/longest_common_subsequence.java
--- 
+++ 
@@ -2,7 +2,7 @@
     if (a.isEmpty() || b.isEmpty()) {
         return "";
     } else if (a.charAt(0) == b.charAt(0)) {
-        return a.charAt(0) + longest_common_subsequence(a.substring(1), b);
+        return a.charAt(0) + longest_common_subsequence(a.substring(1), b.substring(1));
     } else {
         String fst = longest_common_subsequence(a, b.substring(1));
         String snd = longest_common_subsequence(a.substring(1), b);

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/gpt-neox/longest_common_subsequence.py
--- 
+++ 
@@ -3,7 +3,7 @@
         return ''
 
     elif a[0] == b[0]:
-        return a[0] + longest_common_subsequence(a[1:], b)
+        return a[0] + longest_common_subsequence(a[1:], b[1:])
 
     else:
         return max(
